opal-r/src/main/java/org/obiba/opal/web/r/Dtos.java

```diff
@@ -54,4 +56,15 @@ private Dtos() {}
         .build();
   }
 
+
+  public static Ws.ClientErrorDto getErrorMessage(Response.StatusType status,
+                                                  String errorStatus,
+                                                  RuntimeException exception) {
```

Comment:
RuntimeException => REvaluationRuntimeException

Reply:
I left _RuntimeException_ to possibly for other Exception in _web.r_.

Reply:
It's not what I would call any RuntimeException : `OpalR.RServerRuntimeErrorDto.errors`